### PR TITLE
[frontend] mainnet launch (test version)

### PR DESCRIPTION
**Pull request by 0xChqrles**

### CODE REVIEW

File: frontend/src/state/memecoin/updater.tsx [hunk 1]
```diff
@@ -140,6 +143,8 @@ export default function MemecoinUpdater(): null {
 
         const lockManager = res.result[20] as string
 
+        console.log(res.result[21])
```
Comment (enitrat) [inline frontend/src/state/memecoin/updater.tsx:146]: log?

File: frontend/.env.development [hunk 2]
```diff
@@ -1,2 +1,2 @@
 REACT_APP_NETHERMIND_KEY=YUVMGlmfoAEUCrf9vRERnyb7YeCNFE5sA8Awf0htjD2RyKS7
-REACT_APP_NETWORK=goerli
```
Comment (fricoben) [inline frontend/.env.development:2]: this .env file shouldn't be on github

File: frontend/src/state/memecoin/updater.tsx (hunk 1 above)
Comment (enitrat) [reply frontend/src/state/memecoin/updater.tsx:146]:
```suggestion
```

**Review by enitrat — APPROVED** [review]

File: frontend/.env.development (hunk 2 above)
Comment (enitrat) [reply frontend/.env.development:2]: not important as its front

**Pull request merged**